test/integration/associations/has-many.test.js

```diff
@@ -1305,6 +1305,21 @@ describe(Support.getTestDialectTeaser('HasMany'), () => {
       });
     });
 
+    it('should create with nested associated models', function() {
+      const User = this.User,
+        Task = this.Task,
+        values = {
+          username: 'John',
+          email: 'john@example.com',
+          tasks: [{ title: 'Fix PR' }]
+        };
+
+      return User.create(values, { include: ['tasks'] }).then(user =>
+        Task.findOne({ where: { userEmail: user.email } }).then(task => {
```

Comment:
returned `user` itself should contain `tasks`. In any case this test should

1) expect tasks defined for returned `user`
2) try to load tasks for same user with `user.getTasks` so we can make sure relationship was successful

Reply:
Done